src/components/SiteHeader/SiteHeader.js

```diff
@@ -86,6 +86,13 @@ const SiteHeaderBase = kind({
 								>
 									Github
 								</Cell>
+								<Cell
+									component={OutboundLink}
+									shrink
+									href="https://enactjs.com/sampler"
+								>
+									Sampler
```

Comment:
I prefer `UI Components` or `Components`.

Reply:
Fixed it.